env_support/cmsis-pack/lv_os_custom_c.txt

```diff
@@ -36,10 +36,11 @@
  *   GLOBAL FUNCTIONS
  **********************/
 
-lv_result_t lv_thread_init(lv_thread_t * thread, lv_thread_prio_t prio, void (*callback)(void *), size_t stack_size,
+lv_result_t lv_thread_init(lv_thread_t * thread, const char * const pcName, lv_thread_prio_t prio, void (*callback)(void *), size_t stack_size,
                            void * user_data)
 {
     LV_UNUSED(thread);
+    LV_UNUSED(pcName);
```

Comment:
Nit: pcName does not seem to follow the snake case standard even though this file is a custom one.